### PR TITLE
implement as: :raw for watch

### PR DESCRIPTION
**Pull request by grosser**
fixes https://github.com/abonas/kubeclient/issues/282
part of https://github.com/abonas/kubeclient/issues/263

@cben @moolitayer

### CODE REVIEW

File: lib/kubeclient/common.rb [hunk 1]
```diff
@@ -247,6 +247,7 @@ def rest_client
     #   :label_selector (string) - a selector to restrict the list of returned objects by labels.
     #   :field_selector (string) - a selector to restrict the list of returned objects by fields.
     #   :resource_version (string) - shows changes that occur after passed version of a resource.
+    #   :as (symbol) - if :raw, return the raw response body (as a string)
```
Comment (moolitayer) [inline lib/kubeclient/common.rb]: :+1: for taking doc from get_entit* please also include the default clause
```
#   Default response type will return an entity as a  RecursiveOpenStruct
#   (:ros) object, unless `:as` is passed with `:raw`.
```

**Comment by moolitayer** [comment]
Looks good. One nit comment, also please add a README note.

It could be in "Getting raw responses" section that  mentions raw for get_x or in/after "Receive entity updates" that talks about watch. I prefer in the watch section since that is probably what a developer writing watch code would go to.

**Comment by grosser** [comment]
updated, looks good ?

**Comment by moolitayer** [comment]
Labeled v2.x/yes

CC @cben @jhernand

**Pull request merged**